crescent/commands/decorators.py

```diff
@@ -132,7 +134,7 @@ def command(
     return register_command(
         callback=callback_func,
         command_type=CommandType.SLASH,
-        name=name or callback.__name__,
+        name=name.lower() if name is not None else callback.__name__.lower(),
```

Comment:
if the user specifies a name it can not be made lowercase. It would be better to raise an exception if a user specifies an invalid name.

Reply:
Aka a BadRequestError from Hikari's errors from Discord, not handling anything, similar to what we discussed about options? Or in this case, differently?

Reply:
I think its best to leave it as a BadRequestError for now. Im gonna open an issue to handle that error better so it can be PRed in the future.

Reply:
Alright, I'll change it now.